doc/release-notes/10889_bump_PG17_FlyWay10.md

```diff
@@ -0,0 +1,7 @@
+This release bumps both the Postgres JDBC driver and Flyway versions. This should better support Postgres version 17, and as of version 10 Flyway no longer requires a paid subscription to support older versions of Postgres.
+
+While we don't encourage the use of older Postgres versions, this flexibility may benefit some of our long-standing installations in their upgrade paths. Postgres 13 remains the version used with automated testing.
+
+As part of this update, the containerized development environment now uses Postgres 17 instead of 16. Worst case, developers can start with a fresh database, if necessary.
```

Comment:
I was curious what would happen if I do the following:

- Spin up containers on the develop branch, let Dataverse configure itself
- Stop containers (don't delete data)
- Switch to this branch
- Spin up containers

I got these errors:

`dev_dataverse> 2024-11-01T19:06:18Z INF [TCP] Checking the postgres:5432 ...`
`dev_dataverse> 2024-11-01T19:06:18Z ERR Expectation failed error="failed to establish a tcp connection, caused by: dial tcp: lookup postgres on 127.0.0.11:53: no such host"`

Next I'll try blowing the data away.